generators/server/templates/src/main/java/package/domain/_Authority.java

```diff
@@ -18,7 +18,7 @@
 -%>
 package <%=packageName%>.domain;
 
-<% if (hibernateCache !== 'no' && databaseType === 'sql') { %>import org.hibernate.annotations.Cache;
+<% if (cacheProvider !== 'no' && databaseType === 'sql') { %>import org.hibernate.annotations.Cache;
```

Comment:
For all the hibernate related parts the check should be with the new enableHibernateCache variable and we can drop the cache and db check, this will apply to all domain objects/entities

Reply:
I agree with you. I didn't want to change everything at the risk of making a mistake: D
In addition, I'm a little afraid of merge conflict given the number of code impacted currently.

I will not be able to try to do it until Monday

Reply:
thats ok its not urgent anyway